dm_cli/import_package.py

```diff
@@ -43,6 +43,25 @@ def dict(self):
         }
 
 
+from dataclasses import dataclass
+from typing import Literal, NewType
+
+TDependencyProtocol = NewType("TDependencyProtocol", Literal["sys", "http"])
+
+
+@dataclass(frozen=True)
+class Dependency:
+    """Class for any dependencies (external types) a entity references"""
+
+    alias: str
+    # Different ways we support to fetch dependencies.
+    # sys: This DMSS instance
```

Comment:
"This DMSS instance" is a bit confusing.

Maybe what we want to say is that sys refers to the DMSS instance defined with the dmss url in dm-cli (http://localhost:5000 by default)